src/firmware_main_loop.c

```diff
@@ -56,6 +56,13 @@ void firmware_main_loop(void)
     uint8_t uart_write_buf[UART_OUT_BUF_LEN];
     ringbuffer_init(&uart_write_queue, &uart_write_buf, UART_OUT_BUF_LEN);
 
+    /// If the bootloader has booted the BLE chip, it isn't aware of the private name
+    // Send over the private name
```

Comment:
Outdated comment, there is no "private name"

Reply:
fixed